src/Services/SAMLConfiguration.php

```diff
@@ -52,6 +52,25 @@ class SAMLConfiguration
      */
     private static $authn_contexts;
 
+    /**
+     * @config
+     * @var bool Whether or not we expect to receive a binary NameID from the IdP. We expect to receive a binary NameID
+     * from ADFS, but don't expect it from Azure AD or most other SAML implementations that provide GUIDs.
+     *
+     * Defaults to true to preserve backwards compatibility (ADFS).
+     */
+    private static $expect_binary_nameid;
```

Comment:
Docblock states that this value defaults to true, however there is no default value, thus it is false (`NULL`).
If it is true from yml sources I think it best also reflected here in the "hard truth", coinciding with the docblock.